packages/core/admin/ee/server/src/index.ts

```diff
@@ -49,6 +51,12 @@ const getAdminEE = () => {
     async register({ strapi }: { strapi: Core.Strapi }) {
       // Run the the default registration
       await eeAdmin.register({ strapi });
+
+      // Register internal ai service
+      if (isAIEnabled) {
+        strapi.add('ai', createAIService({ strapi }));
```

Comment:
I think what this does is different from creating a service, it's more creating an entire namespace outside of services. Is it intentional? If so I'd rename createAIService and move it out of services folder

Reply:
I'm fine with the non-service approach, just find the service name misleading in this case

Reply:
The `add` / `get` method is apparently part of the Container interface (according to windsurf's AI wiki 🙃), wdyt about `createAIContainer`? I should probably move the function into a folder called containers then though 🤔 

<img width="496" height="595" alt="Screenshot 2025-11-03 at 11 39 57" src="https://github.com/user-attachments/assets/3f0a7fa8-5e12-44f3-a6ee-f639548661fd" />

Reply:
that would work 👍